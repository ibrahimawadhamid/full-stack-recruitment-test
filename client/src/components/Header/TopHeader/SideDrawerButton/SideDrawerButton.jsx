import React, {Component} from 'react';

import BpkButton from 'bpk-component-button';
import BpkMenuIcon from 'bpk-component-icon/lg/menu';
import {withLargeButtonAlignment} from 'bpk-component-icon';

const AlignedMenuIcon = withLargeButtonAlignment(BpkMenuIcon)

class SideDrawerButton extends Component {
  render() {
    return (
      <BpkButton link>
        <AlignedMenuIcon/>
      </BpkButton>
    );
  }
};

export default SideDrawerButton;
