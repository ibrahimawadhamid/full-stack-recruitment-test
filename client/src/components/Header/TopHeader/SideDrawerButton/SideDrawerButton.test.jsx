import React from 'react';
import TestRenderer from 'react-test-renderer';

import SideDrawerButton from './SideDrawerButton';

describe('SideDrawerButton', () => {
    it('should render correctly', () => {
        const tree = TestRenderer.create(<SideDrawerButton />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});