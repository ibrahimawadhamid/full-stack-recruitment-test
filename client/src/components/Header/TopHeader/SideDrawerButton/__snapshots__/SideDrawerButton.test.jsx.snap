// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideDrawerButton should render correctly 1`] = `
<button
  className="bpk-button bpk-button--link"
  disabled={false}
  onClick={null}
  type="button"
>
  <span
    style={
      Object {
        "display": "inline-block",
        "lineHeight": "1.5rem",
        "marginTop": "0.1875rem",
        "verticalAlign": "top",
      }
    }
  >
    <svg
      height="24"
      style={
        Object {
          "height": "1.5rem",
          "width": "1.5rem",
        }
      }
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20 8H4c-.6 0-1-.4-1-1s.4-1 1-1h16c.6 0 1 .4 1 1s-.4 1-1 1zm1 4c0-.6-.4-1-1-1H4c-.6 0-1 .4-1 1s.4 1 1 1h16c.6 0 1-.4 1-1zm0 5c0-.6-.4-1-1-1H4c-.6 0-1 .4-1 1s.4 1 1 1h16c.6 0 1-.4 1-1z"
      />
    </svg>
  </span>
</button>
`;
